src/mmw/js/src/analyze/templates/streamTable.html

```diff
@@ -7,7 +7,7 @@
             <th class="text-right" data-field="lengthkm" data-sortable="true" data-sorter="window.numericSort">
                 Total Length (km)
             </th>
-            <th class="text-right" data-field="slopepct" data-sortable="true" data-sorter="window.numericSort">
+            <th class="text-right" data-field="avgslope" data-sortable="true" data-sorter="window.numericSort">
```

Comment:
Since there are potentially some `noData` values here, wonder if we should switch to use `window.noDataSort` for this column.

Reply:
Good idea!